lib/generators/templates/atmosphere.rb

```diff
@@ -117,4 +117,12 @@
   # this new class name as `ability_class` parameter value.
   #
   # config.ability_class = 'Atmosphere::Ability'
+
+  # Password for vm instantiated at Azure (ssh keys are not suported)
+  # By default azure_vm_password should be defined in air.yml
+  # config.azure_vm_password = Settings.azure_vm_password
```

Comment:
@nowakowski we don't have `air.yml.example` in atmosphere (it is vphshare specific). What is more we don't have `Settings` class in atmosphere. Please add here some example text.